### PR TITLE
Home page responsive

### PR DESCRIPTION
**Pull request by jcfilben**
Make the cards on the home page and the 'What are grommet Tools?' scale better

### CODE REVIEW

File: src/pages/Home/Home.js [hunk 1]
```diff
@@ -189,14 +189,17 @@ with Grommet and the Grommet Designer."
...
         bottom: 'none',
       }}
-      direction="row"
+      direction="row-responsive"
+      align="end"
+      gap="xlarge"
     >
-      <Box width="medium" margin={{ bottom: 'xlarge' }}>
-        <Heading>What are grommet Tools?</Heading>
+      <Box alignSelf="center" size="large" margin={{ bottom: 'xlarge' }}>
```
Comment (ShimiSun) [inline src/pages/Home/Home.js:199]: there is no prop of `size` to Box.

File: src/pages/Home/Home.js [hunk 2]
```diff
@@ -189,14 +189,17 @@ with Grommet and the Grommet Designer."
...
+        <Heading level={2} size="xlarge">
```
Comment (ShimiSun) [inline src/pages/Home/Home.js:200]: Can you confirm you have a Level 1 heading before applying the second?

File: src/pages/Home/Home.js [hunk 3]
```diff
@@ -227,7 +230,14 @@ with Grommet and the Grommet Designer."
           />
         </Box>
       </Box>
-      <Image src="RockinGremlin.svg" fit="contain" />
+      <Box size="medium">
```
Comment (ShimiSun) [inline src/pages/Home/Home.js:233]: No `size` prop to Box.

File: src/theme.js [hunk 4]
```diff
@@ -9,7 +9,6 @@ export const grommetToolsTheme = deepMerge(grommet, {
       none: '0px',
     },
     size: {
-      cards: '500px',
```
Comment (ShimiSun) [inline src/theme.js:12]: Nice clean-up!!!

File: src/pages/Home/Home.js (hunk 1 above)
Comment (jcfilben) [reply src/pages/Home/Home.js]: Removed `size` prop

File: src/pages/Home/Home.js (hunk 2 above)
Comment (jcfilben) [reply src/pages/Home/Home.js:214]: Added level 1 heading to 'Grommet Tools' at the top of the page and removed custom headings from theme

File: src/pages/Home/Home.js (hunk 3 above)
Comment (jcfilben) [reply src/pages/Home/Home.js]: Removed `size` prop

**Comment by jcfilben** [comment]
Also added `a11yTitle` to images

**Pull request merged**